pkg/ocm/client.go

```diff
@@ -77,6 +78,10 @@ func NewClient(config Config, log logrus.FieldLogger) (*Client, error) {
 		logger: logger,
 		Cache:  cache.New(1*time.Minute, 30*time.Minute),
 	}
+	err := client.newConnection()
+	if err != nil {
+		return nil, fmt.Errorf("Unable to build OCM connection: %s", err.Error())
```

Comment:
Why not errors.Wrap? i think we mostly use it.

Reply:
Done